test-data/packages/typedpkg-stubs/setup.py

```diff
@@ -0,0 +1,13 @@
+"""
```

Comment:
I can't add a comment to an empty file -- IIUC the py.typed file shouldn't be needed here?